unitTests/ui/FieldTableTest.ts

```diff
@@ -95,8 +95,11 @@ export function FieldTableTest() {
           expect(findToggleButton()).not.toBeNull();
         });
 
-        it('shows a toggle caption set to the minimizedTitle', () => {
-          expect(findToggleCaption().textContent).toBe(test.cmp.options.minimizedTitle);
+        it('shows a toggle caption set to the minimizedTitle', done => {
+          requestAnimationFrame(() => {
+            expect(findToggleCaption().textContent).toBe(test.cmp.options.minimizedTitle);
+            done();
+          });
```

Comment:
Just to make sure, does this test have the necessary setup to reproduce the bug? In that if `requestAnimationFrame` is removed from source, it will fail this test?

Also, is the test asynchronous, or can the `done` be removed?

Reply:
Yeah, if you remove this the test fails.

And it's executed inside a callback function that gets executed "sometimes when the browser decides to", so it is async, yes.